frontend/pages/ManageControlsPage/MacOSSettings/MacOSSettings.tsx

```diff
@@ -67,9 +79,7 @@ const MacOSSettings = ({
           <CurrentCard
             key={teamId}
             currentTeamId={teamId}
-            profiles={profiles}
-            onProfileUpload={refectchProfiles}
-            onProfileDelete={refectchProfiles}
+            onDataChange={refetchAggregateProfileStatus}
```

Comment:
What do you think about calling this `onMutation` to signify that it is more of an end-user driven event rather than reacting to some upstream data change?

Reply:
agreed. i'll rename